src/controller/python/OpCredsBinding.cpp

```diff
@@ -139,8 +139,12 @@ ChipError::StorageType pychip_OpCreds_AllocateController(OpCredsContext * contex
     VerifyOrReturnError(devCtrl != nullptr, CHIP_ERROR_NO_MEMORY.AsInteger());
 
     // Initialize device attestation verifier
-    // TODO: Replace testingRootStore with a AttestationTrustStore that has the necessary official PAA roots available
+#ifdef MATTER_FILE_SYSTEM_ENABLED
+    const chip::Credentials::AttestationTrustStore * testingRootStore =
+        chip::Credentials::GetTestFileAttestationTrustStore("development_trust_store.paa");
```

Comment:
This model is problematic because it relies on the tools being run from a single place rather than letter the caller of a tool provide the correct root paths